hermit/ui/common.py

```diff
@@ -4,6 +4,7 @@
 import hermit.ui.state as state
 import traceback
 import sys
+import hermit
```

Comment:
because of the `import *` in the `__init__` file, I think this is going to load most of the hermit code
Is it possible to just import the version?

Reply:
True, everything in hermit will be available as hermit.everything. It doesn't seem to create a dependency loop, though. I'm happy to change it to just import what I need.